src/shared/components/download/Download.tsx

```diff
@@ -102,6 +116,47 @@ const Download = (props: DownloadProps<JobTypes>) => {
     { props, job, selectedColumns: columnNames },
     getDownloadInitialState
   );
+  const { data } = useDataApi<ReceivedFieldData>(
+    apiUrls.resultsFields(namespace)
+  );
+
+  const fieldData = useMemo(
+    () => prepareFieldData(data, undefined, true),
+    [data]
+  );
+
+  const getFieldsWithMultipleValueXref = useCallback(
+    (fieldData: FieldData | FieldDatum, id: string): boolean | undefined => {
+      if (Array.isArray(fieldData)) {
+        for (const item of fieldData) {
+          const isFull = getFieldsWithMultipleValueXref(item, id);
+          if (isFull) {
+            return true;
+          }
+        }
+      } else if (fieldData.items) {
+        return getFieldsWithMultipleValueXref(fieldData.items, id);
+      } else if (fieldData.id === id) {
+        return fieldData.addAsterisk;
+      }
+      return undefined;
```

Comment:
```suggestion
      return true;
```
I think this would allow the function to be a fully boolean return.